config/v1/0000_10_config-operator_01_infrastructure.crd.yaml

```diff
@@ -185,6 +197,21 @@ spec:
                   a human friendly name. Once set it should not be changed. Must be
                   of max length 27 and must have only alphanumeric or hyphen characters.
                 type: string
+              infrastructureTopology:
+                description: InfrastructureTopology expresses the expectations for
+                  infrastructure services that do not run on control plane nodes,
+                  usually indicated by a node selector for a `role` value other than
+                  `master`. This includes, but is not necessarily limited to, the
```

Comment:
I think we can stop with the period after "master" here, and leave the next 2 sentences out of the CRD definition.

Reply:
done